pkg/lib/files/files.go

```diff
@@ -456,7 +456,7 @@ func IsFilePathPython(path string) bool {
 type IgnoreFn func(string, os.FileInfo) (bool, error)
 
 func IgnoreHiddenFiles(path string, fi os.FileInfo) (bool, error) {
```

Comment:
What do you think of renaming it to `IgnoreHiddenFilesExceptDotEnv` instead? Otherwise, I think, it would be trickier to remove the ignored file (`.env`) from `IgnoreHiddenFiles` if it were to be separate.

I've pushed a commit to address this.